### PR TITLE
[Doc Improvement][Update doc site for prerequisite-checker][2425966]

### PR DESCRIPTION
**Pull request by v-ksar**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c286aeb4694495871bad091390091e59b6ce05ef/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6b51ee93-30ec-4cef-9cd5-73c831db8f1f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/64dd8c521da178fde557c0103f0fa283e52f8280/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7b639e82-05aa-4b11-91e9-3aa695c62f2c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-background-process.md](https://github.com/MicrosoftDocs/msteams-docs/blob/64dd8c521da178fde557c0103f0fa283e52f8280/msteams-platform%2Ftoolkit%2Fdebug-background-process.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da9a6617-d813-4c5d-bdb1-da78343eabc6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/951f6b223d9ca043db7c2b64f4e5352aa3c84f96/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/42d27916-5eb6-4afd-b16b-0f2d6302b474) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-nodejs.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/951f6b223d9ca043db7c2b64f4e5352aa3c84f96/msteams-platform%2Fsbs-gs-nodejs.yml) | 91 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ec8a9d6b-dab5-462f-a4f0-95fd473f7c97) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/951f6b223d9ca043db7c2b64f4e5352aa3c84f96/msteams-platform%2Fsbs-gs-spfx.yml) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/059f27ed-2394-4698-9a2d-a1effc89e59b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/tab-requirements.md](https://github.com/MicrosoftDocs/msteams-docs/blob/951f6b223d9ca043db7c2b64f4e5352aa3c84f96/msteams-platform%2Ftabs%2Fhow-to%2Ftab-requirements.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ba8e61f-da4f-406c-9e79-cfa4730a5c1f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-background-process.md](https://github.com/MicrosoftDocs/msteams-docs/blob/951f6b223d9ca043db7c2b64f4e5352aa3c84f96/msteams-platform%2Ftoolkit%2Fdebug-background-process.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7f6d5513-d50c-409e-9f29-228a4053676d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3f42658f05d1640a67a50fb7b0ecbab860f5691e/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54efe264-07b3-47b8-924b-597217faec03) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-nodejs.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3f42658f05d1640a67a50fb7b0ecbab860f5691e/msteams-platform%2Fsbs-gs-nodejs.yml) | 91 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7da4f12a-8e59-49eb-a9d6-71823f45982d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/3f42658f05d1640a67a50fb7b0ecbab860f5691e/msteams-platform%2Fsbs-gs-spfx.yml) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e65c0354-b402-4821-a92b-8a4d6b860e2c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/tab-requirements.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3f42658f05d1640a67a50fb7b0ecbab860f5691e/msteams-platform%2Ftabs%2Fhow-to%2Ftab-requirements.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c9cb4a41-2471-4ffb-bdff-42b443a6d134) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-background-process.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3f42658f05d1640a67a50fb7b0ecbab860f5691e/msteams-platform%2Ftoolkit%2Fdebug-background-process.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21a32a73-e83b-49a4-b382-db1ab76e6d7c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-spfx.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ items:
     | &nbsp; | [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
     | &nbsp; | [Visual Studio Code](https://code.visualstudio.com/download) | JavaScript, TypeScript, or SharePoint Framework (SPFx) build environments. Use version 1.55 or later. |
     | &nbsp; | [Teams Toolkit](#install-the-teams-toolkit) | A Visual Studio Code extension that creates a project scaffolding for your app. Use latest version. |
-    | &nbsp; | [Node.js](https://nodejs.org/en/download/) | Back-end JavaScript runtime environment. Use the Node.js versions 10, 12, or 14 LTS release. |
+    | &nbsp; | [Node.js](https://nodejs.org/en/download/) | Back-end JavaScript runtime environment. Use the Node.js version 16 LTS release. |
```
Comment (zyxiaoyuer) [inline msteams-platform/sbs-gs-spfx.yml:40]: For SPFx, use NodeJs version 12, or 14 LTS release.

**Review by zyxiaoyuer — CHANGES_REQUESTED** [review]
need update sbs-gs-spfx.yml. For SPFx, use NodeJs version 12, or 14 LTS release.

File: msteams-platform/sbs-gs-spfx.yml (hunk 1 above)
Comment (v-abirade) [reply msteams-platform/sbs-gs-spfx.yml:40]: @v-ksar, please update asap and revert in the Tools doc sync group.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1abbc41309323829eea32ddc6e860a03458da632/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84135a8a-d5ca-4568-9e59-bdd64fb94a3f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-nodejs.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1abbc41309323829eea32ddc6e860a03458da632/msteams-platform%2Fsbs-gs-nodejs.yml) | 91 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38b6821b-9ead-4983-aeb2-ef72f85d49b1) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1abbc41309323829eea32ddc6e860a03458da632/msteams-platform%2Fsbs-gs-spfx.yml) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f536c952-7fd8-4137-9cdd-200a07522e5e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/tab-requirements.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1abbc41309323829eea32ddc6e860a03458da632/msteams-platform%2Ftabs%2Fhow-to%2Ftab-requirements.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ab47bbaa-d2b9-427c-9506-9de7304761cc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-background-process.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1abbc41309323829eea32ddc6e860a03458da632/msteams-platform%2Ftoolkit%2Fdebug-background-process.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9e7cf9dc-712c-4810-aa0e-52eaf92a42c3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-gs-spfx.yml (hunk 1 above)
Comment (v-ksar) [reply msteams-platform/sbs-gs-spfx.yml]: Updated as mentioned

**Review by v-abirade — APPROVED** [review]
Approved

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/sbs-gs-javascript.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fb38f388052be090570b172b935226c0ea53c77f/msteams-platform%2Fsbs-gs-javascript.yml) | 92 | 78 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84817dac-aa00-4dfa-818b-f41f8cf6f728) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-nodejs.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fb38f388052be090570b172b935226c0ea53c77f/msteams-platform%2Fsbs-gs-nodejs.yml) | 91 | 29 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3c04bbbd-2577-4cd9-84fb-48b18dc7839c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-gs-spfx.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/fb38f388052be090570b172b935226c0ea53c77f/msteams-platform%2Fsbs-gs-spfx.yml) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/929f5fdc-5f1f-4a6a-954e-06a9ea2cbb11) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/tab-requirements.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb38f388052be090570b172b935226c0ea53c77f/msteams-platform%2Ftabs%2Fhow-to%2Ftab-requirements.md) | 91 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/428ec04e-d09b-47d4-bd06-e5b2d774f88f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/toolkit/debug-background-process.md](https://github.com/MicrosoftDocs/msteams-docs/blob/fb38f388052be090570b172b935226c0ea53c77f/msteams-platform%2Ftoolkit%2Fdebug-background-process.md) | 85 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd14cda6-2a5b-487f-b493-a0f4e2ba9458) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — APPROVED** [review]
Approve

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**